data/sql/updates/pending_db_world/rev_1645624680895656000.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1645624680895656000');
+
+-- Add gossip flag to Expedition Commander, previously was 0
+UPDATE creature_template SET npcflag = 1 WHERE entry IN (33210,34254);
```

Comment:
```suggestion
UPDATE `creature_template` SET `npcflag` = `npcflag` | 1 WHERE `entry` IN (33210,34254);
```

It is prefer to set the flag this way, just in case there are other change that also will affect this flag. This will ensure only that the correct bit is set.

Reply:
Thanks!